pkgs/development/python-modules/pyosf/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, pytestrunner
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pyosf";
+  version = "1.0.5";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "psychopy";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fkpmylpcbqa9ky111mz4qr1n8pik49gs7pblbb5qx6b54fzl5k2";
+  };
+
+  preBuild = "export HOME=$TMP";
+  buildInputs = [ pytestrunner ];
```

Comment:
Tests are disabled, why is this needed?

Reply:
Because the author has put it in `setup_requires` ...

Reply:
I have added a comment to this effect inline.